src/Exception/NotFoundException.php

```diff
@@ -2,11 +2,13 @@
 
 namespace Http\Discovery\Exception;
 
+use Http\Discovery\Exception;
```

Comment:
I followed the example in HTTPlug: https://github.com/php-http/httplug/tree/master/src

I did some (very little) research and it seams that (a few) others are doing the same.